lib/musiclibrarycontroller.rb

```diff
@@ -22,21 +14,16 @@ def call
   end
 
   def run_command(command)
-    if @menu.keys.include?(command)
-      send(@menu[command])
-      call
-    elsif command != "exit"
-      Message.invalid
+    if command != "exit"
+      Message.invalid unless send(command.tr(" ", "_")) 
       call
-    else
-      return
     end
   end
 
   def list_songs
-    @control.files.each.with_index(1) do |file, index|
-      puts "#{index}. #{file.chomp('.mp3')}"
```

Comment:
Defence feedback: Avoid duplicating ".chomp('.mp3')". It has been used in another class